.chloggen/influxdb.yaml

```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: new_component
```

Comment:
I think new component is used if you create brand new type in registry and in this case  it should be enhancement

@joaopgrassi could you confirm if it's correct?

Reply:
> I think new component is used if you create brand new type in registry and in this case it should be enhancement
> 
> @joaopgrassi could you confirm if it's correct?

I finished the changelog according to https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#adding-a-changelog-entry
It doesn't look like it contains the distinction between `new_component` and `enhancement`. At first, I though this is a new framework so choose `new_component`. If we discuss the difference between them this time, I think it had better to add some explanation about them in document later：）

Reply:
> new_component is a new namespace, in this case this is extension of existing db.

Got it, I modified it now.